iberian_labs.md

```diff
@@ -4,3 +4,90 @@ title: Iberian Labs
+
+Main research topics: *cognitive neuroscience, neurodegenerative disorders, neurodevelopmental disorders, neuropsychiatric disorders*
+
+Contact: Miguel Castelo-Branco, mcbranco@fmed.uc.pt
+
+Website: [https://www.uc.pt/en/uid/cibit/](https://www.uc.pt/en/uid/cibit/)
+***
+* ### Image Processing Lab (LPI)
+University of Valladolid
+
+Castilla y León, Spain
+
+**LPI** is an officially recognized research group by the University of Valladolid and has been considered by the regional government as a Group of Excellence. Its main activity is focused on medical image analysis and biomedical signal processing.
+
+Main research topics: *diffusion MRI, acquisition, image processing, cardiac MRI*
+
+Contact: Carlos Alberola, contact@lpi.tel.uva.es
+
+Website: [http://www.lpi.tel.uva.es/](http://www.lpi.tel.uva.es/)
+***
+* ### Proaction Lab
+University of Coimbra
+
+Coimbra, Portugal
+
+The **Proaction Lab** focuses on Cognitive Neuroscience approaches to unravel the processes associated with the perception and recognition of objects and actions, as well as how knowledge and information are stored and organized in the brain.
+
+Main research topics: *perception, objects, mind wandering, collaborative robots*
+
+Contact: Jorge Almeida, jorgealmeida@fpce.uc.pt
+
+Website: [http://proactionlab.fpce.uc.pt/](http://proactionlab.fpce.uc.pt/)
+
+Twitter: [@proactionlab](https://twitter.com/proactionlab)
+***
+* ### Signal Processing in Neuroimaging (SPIN)
+Basque Center on Cognition, Brain and Language
+
+Donostia - San Sebastián, Spain 
+
+The **SPIN** lab investigates methods and develops novel algorithms for acquisition and analysis of neuroimaging (mostly, MRI) and biomedical data to advance our understanding of the function, structure and physiology in the healthy and diseased human brain.
+
+Main research topics: *biomedical signal processing, functional MRI acquisition and analysis, calibrated fMRI, MRI*
+
+Contact: Cesar Caballero-Gaudes, c.caballero@bcbl.eu
+
+Website: [https://www.bcbl.eu/en/research/research-groups/signal-processing-neuroimaging/](https://www.bcbl.eu/en/research/research-groups/signal-processing-neuroimaging/)
+
+***
+## Iberians Around the World
+* ### Quantitative Imaging Methods Lab
+University of Wisconsin-Madison
+
+Madison, Wisconsin, USA
+
+The **Quantitative Imaging Methods Lab** develops new techniques, particularly for quantitative body MRI, and works with physicists, engineers, basic scientists, and clinicians, to validate and apply these techniques.
+
+Main research topics: *quantitative, diffusion, susceptibility, iron, fat*
+
+Contact: Diego Hernando, dhernando@wisc.edu
+
+Website: [https://qiml.radiology.wisc.edu/](https://qiml.radiology.wisc.edu/)
+
+Twitter: [@dherarr](https://twitter.com/dherarr)
```

Comment:
This should go on the Iberian Around the World page. You're gonna have to add it to `iberians_around_world.md` 😉

Reply:
Sure. Thanks for reminding.